docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -277,24 +407,74 @@ The `locale` prop of the `LocalizationProvider` component have been renamed `ada
  </LocalizationProvider
 ```
 
-## Component slots (custom sub-components)
+## Component slots / component props slots
+
+All the props used to pass props to parts of the UI (e.g: pass a prop to the input) have been replaced by components props slots.
+All the props used to override parts of the UI (e.g: pass a custom day renderer) have been replaced by components slots.
+
+You can find more information about this pattern in the [MUI Base documentation](https://mui.com/base/getting-started/usage/#shared-props).
+
+There changes always apply to all the components that had the prop.
+For example, the `ToolbarComponent` has been replaced by a `Toolbar` component slot on all pickers.
+
+### Input renderer (required in v5)
+
+- The `renderInput` has been replaced by an `input` component props slot:
+
+  ```diff
+   <DatePicker
+  -  renderInput={(inputProps) => <TextField {...props} variant="outlined" />}
+  +  componentsProps={{ input: { variant: 'outlined' }}
+   />
+
+   <DateRangePicker
+  -  renderInput={(startProps, endProps) => (
+  -    <React.Fragment>
+  -      <TextField {...startProps} variant="outlined" />
+  -      <Box sx={{ mx: 2 }}> - </Box>
+  -      <TextField {...endProps} variant="outlined" />
+  -    </React.Fragment>
+  -  )}
+  +  componentsProps={{ input: { variant: 'outlined' }}
+   />
+  ```
+
+  The Date Range Picker also have a new `fieldSeparator` component slot and component props slot to customize only this part of the UI:
```

Comment:
Either this should also be a sub-section (list item), or we need to drop it from the top paragraph.

```suggestion
- The Date Range Picker also have a new `fieldSeparator` component slot and component props slot to customize only this part of the UI:
```